### PR TITLE
Check level is not opened when running 'ed_physxUpdatePrefabsWithColliderComponents' command

### LINKED ISSUE #14656: The ed_physxUpdatePrefabsWithColliderComponents command can invisibly corrupt level files.
**Describe the bug**
When running the ed_physxUpdatePrefabsWithColliderComponents command on a level that contains prefabs with Collider components in it, the command appears to generate "add" patches in the level that can't be viewed in the Editor, but overrides whatever Collider data is in the prefab, preventing changes to the prefab from working.

**Steps to reproduce**
1. Create a level
2. Add an entity named "PhysicsEntity" with a PhysX Collider component and a Static Rigid Body.
3. Right-click on the entity and make it a prefab. Save the prefab in the level folder.
4. Save the level
5. Close the Editor
6. Manually edit the PhysicsEntity.prefab in a text editor to remove the EditorStaticRigidBodyComponent
7. Restart the Editor
**8. Load the created level** - this goes against the "important notice" instructions, but it's needed to recreate the bug.
9. Run the "ed_physxUpdatePrefabsWithColliderComponents" command.
10. Save the level and the prefab.
11. Examine the level prefab file.

**Expected behavior**
The level prefab file should remain untouched.

**Actual behavior**
The level prefab gets a bunch of "add" patches added to it that silently start affecting how the level works and what happens when you edit the prefab. It can lead to a lot of subtle bugs in which physics stops working correctly.

**Found in Branch**
development

**Commit ID from [o3de/o3de](https://github.com/o3de/o3de) Repository**
5ba0c190f738d5d9c1f148ab1fd83a99cfa45b7b

**Desktop/Device (please complete the following information):**
PC

NOTE: This was discovered on MultiplayerSample after painstakingly investigating why physics on the teleporters stopped working, even though everything about them appeared correct in the teleporter prefab and the Editor.

### PR DESCRIPTION
**Pull request by moraaar**
## What does this PR do?

Fixes #14656

## How was this PR tested?

Run the command with and without a level opened in the editor.

### CODE REVIEW

File: Gems/PhysX/Code/Editor/Source/Components/Conversion/CollidersPrefabConversion.cpp [hunk 1]
```diff
@@ -127,6 +128,16 @@ namespace PhysX
             return;
         }
 
+        bool isLevelOpen = false;
+        AzToolsFramework::EditorRequests::Bus::BroadcastResult(
+            isLevelOpen, &AzToolsFramework::EditorRequests::Bus::Events::IsLevelDocumentOpen);
+        if (isLevelOpen)
+        {
+            AZ_TracePrintf("PhysXColliderConversion", "There is a level currently opened in the editor. To run this command please restart the editor and run it before opening any level.\n");
```
Comment (SergeyAMZN) [inline Gems/PhysX/Code/Editor/Source/Components/Conversion/CollidersPrefabConversion.cpp:136]: Maybe worth making it a warning or error to make it more visible in the console that the command failed?

**Review by SergeyAMZN — APPROVED** [review]

File: Gems/PhysX/Code/Editor/Source/Components/Conversion/CollidersPrefabConversion.cpp (hunk 1 above)
Comment (moraaar) [reply Gems/PhysX/Code/Editor/Source/Components/Conversion/CollidersPrefabConversion.cpp:136]: I didn't put it for the message when prefabs is disabled... which is a similar case, there is nothing wrong about it, but I get what you mean. Probably not worth another AR run though :)

Comment (moraaar) [reply Gems/PhysX/Code/Editor/Source/Components/Conversion/CollidersPrefabConversion.cpp:136]: I'll change it for a warning.

**Review by greerdv — APPROVED** [review]
Agree with adding the warning, but otherwise looks good to me

**Comment by moraaar** [comment]
> Agree with adding the warning, but otherwise looks good to me

I'll change it for a warning.

**Pull request merged**